diff -ur elmo-0.6/configure.in elmo-0.6-/configure.in
--- elmo-0.6/configure.in	Mon Apr  7 21:26:15 2003
+++ elmo-0.6-/configure.in	Tue Apr  8 18:39:52 2003
@@ -15,7 +15,7 @@
 
 
 dnl Checks for header files.
-AC_CHECK_HEADERS(getopt.h wordexp.h glob.h ncurses.h curses.h)
+AC_CHECK_HEADERS(getopt.h wordexp.h glob.h ncurses/ncurses.h ncurses.h curses.h)
 
 
 dnl Checks for typedefs, structures, and compiler characteristics.
diff -ur elmo-0.6/src/choose.c elmo-0.6-/src/choose.c
--- elmo-0.6/src/choose.c	Mon Apr  7 17:14:56 2003
+++ elmo-0.6-/src/choose.c	Tue Apr  8 18:40:03 2003
@@ -32,6 +32,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/clock.c elmo-0.6-/src/clock.c
--- elmo-0.6/src/clock.c	Mon Mar 17 11:28:37 2003
+++ elmo-0.6-/src/clock.c	Tue Apr  8 18:40:03 2003
@@ -40,6 +40,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/cmd.c elmo-0.6-/src/cmd.c
--- elmo-0.6/src/cmd.c	Mon Apr  7 12:31:04 2003
+++ elmo-0.6-/src/cmd.c	Tue Apr  8 18:40:03 2003
@@ -34,6 +34,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/cmd.h elmo-0.6-/src/cmd.h
--- elmo-0.6/src/cmd.h	Mon Apr  7 12:31:18 2003
+++ elmo-0.6-/src/cmd.h	Tue Apr  8 18:40:03 2003
@@ -30,6 +30,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/compose.c elmo-0.6-/src/compose.c
--- elmo-0.6/src/compose.c	Mon Apr  7 21:09:03 2003
+++ elmo-0.6-/src/compose.c	Tue Apr  8 18:40:03 2003
@@ -45,6 +45,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/elmo.c elmo-0.6-/src/elmo.c
--- elmo-0.6/src/elmo.c	Mon Apr  7 20:16:27 2003
+++ elmo-0.6-/src/elmo.c	Tue Apr  8 18:40:03 2003
@@ -36,6 +36,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/error.c elmo-0.6-/src/error.c
--- elmo-0.6/src/error.c	Sun Mar 30 14:13:16 2003
+++ elmo-0.6-/src/error.c	Tue Apr  8 18:40:03 2003
@@ -45,6 +45,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/fetch.c elmo-0.6-/src/fetch.c
--- elmo-0.6/src/fetch.c	Mon Apr  7 16:58:33 2003
+++ elmo-0.6-/src/fetch.c	Tue Apr  8 18:40:03 2003
@@ -40,6 +40,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/folder.c elmo-0.6-/src/folder.c
--- elmo-0.6/src/folder.c	Mon Apr  7 20:34:20 2003
+++ elmo-0.6-/src/folder.c	Tue Apr  8 18:40:03 2003
@@ -34,6 +34,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/linech.h elmo-0.6-/src/linech.h
--- elmo-0.6/src/linech.h	Fri Apr  4 21:57:57 2003
+++ elmo-0.6-/src/linech.h	Tue Apr  8 18:40:03 2003
@@ -30,6 +30,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/mail.h elmo-0.6-/src/mail.h
--- elmo-0.6/src/mail.h	Mon Mar 17 11:30:48 2003
+++ elmo-0.6-/src/mail.h	Tue Apr  8 18:40:03 2003
@@ -32,6 +32,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/mailinfo.c elmo-0.6-/src/mailinfo.c
--- elmo-0.6/src/mailinfo.c	Mon Mar 17 11:31:22 2003
+++ elmo-0.6-/src/mailinfo.c	Tue Apr  8 18:40:03 2003
@@ -32,6 +32,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/mailreader.c elmo-0.6-/src/mailreader.c
--- elmo-0.6/src/mailreader.c	Mon Mar 31 00:43:24 2003
+++ elmo-0.6-/src/mailreader.c	Tue Apr  8 18:40:03 2003
@@ -36,6 +36,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/progress.c elmo-0.6-/src/progress.c
--- elmo-0.6/src/progress.c	Mon Mar 17 11:32:13 2003
+++ elmo-0.6-/src/progress.c	Tue Apr  8 18:40:03 2003
@@ -35,6 +35,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/rchtype.c elmo-0.6-/src/rchtype.c
--- elmo-0.6/src/rchtype.c	Sat Mar 29 14:18:30 2003
+++ elmo-0.6-/src/rchtype.c	Tue Apr  8 18:40:03 2003
@@ -35,6 +35,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/rchtype.h elmo-0.6-/src/rchtype.h
--- elmo-0.6/src/rchtype.h	Mon Mar 17 17:01:15 2003
+++ elmo-0.6-/src/rchtype.h	Tue Apr  8 18:40:03 2003
@@ -36,6 +36,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/read.c elmo-0.6-/src/read.c
--- elmo-0.6/src/read.c	Mon Apr  7 17:55:24 2003
+++ elmo-0.6-/src/read.c	Tue Apr  8 18:40:03 2003
@@ -37,6 +37,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/screen.c elmo-0.6-/src/screen.c
--- elmo-0.6/src/screen.c	Mon Mar 17 11:35:43 2003
+++ elmo-0.6-/src/screen.c	Tue Apr  8 18:40:03 2003
@@ -34,6 +34,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/screen.h elmo-0.6-/src/screen.h
--- elmo-0.6/src/screen.h	Mon Mar 17 11:36:12 2003
+++ elmo-0.6-/src/screen.h	Tue Apr  8 18:40:03 2003
@@ -30,6 +30,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/select.c elmo-0.6-/src/select.c
--- elmo-0.6/src/select.c	Mon Apr  7 16:44:24 2003
+++ elmo-0.6-/src/select.c	Tue Apr  8 18:40:03 2003
@@ -32,6 +32,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/select.h elmo-0.6-/src/select.h
--- elmo-0.6/src/select.h	Mon Apr  7 16:45:02 2003
+++ elmo-0.6-/src/select.h	Tue Apr  8 18:40:03 2003
@@ -30,6 +30,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
diff -ur elmo-0.6/src/status.c elmo-0.6-/src/status.c
--- elmo-0.6/src/status.c	Mon Mar 17 11:37:17 2003
+++ elmo-0.6-/src/status.c	Tue Apr  8 18:40:03 2003
@@ -34,6 +34,8 @@
 
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif HAVE_NCURSES_NCURSES_H
+# include <ncurses/ncurses.h>
 #elif HAVE_CURSES_H
 # include <curses.h>
 #endif
