diff -Nur elmo-1.0.4.orig/configure.in elmo-1.0.4/configure.in
--- elmo-1.0.4.orig/configure.in	2004-01-29 17:03:22.000000000 +0100
+++ elmo-1.0.4/configure.in	2004-03-01 17:34:35.000000000 +0100
@@ -17,6 +17,7 @@
 
 
 AC_SUBST(HAVE_SHORT_NAME, "")
+AC_SUBST(ELMOLIBS, "")
 
 AC_INIT(elmo, 1.0.4)
 AC_CONFIG_SRCDIR(src/elmo.c)
@@ -116,6 +117,38 @@
 dnl
 AM_CONDITIONAL([EKG_SCANDIR], [test "$have_scandir" = no])
 
+dnl
+dnl home-etc
+dnl
+AC_ARG_WITH(home-etc, [  --with-home-etc[=DIR]          Use HOME-ETC library to let the configuration files be located in a non-standard place ],
+        [
+        if test "$with_home_etc" != "no"
+        then
+          if test "$with_home_etc" != "yes"
+          then
+            LDFLAGS="$LDFLAGS -L$with_home_etc/lib"
+          fi
+
+          saved_LIBS="$LIBS"
+
+          AC_SEARCH_LIBS(get_home_etc_static, home_etc)
+	   AC_CHECK_HEADERS(home_etc.h)
+
+          ELMOLIBS="$ELMOLIBS -lhome_etc"
+          LIBS="$saved_LIBS"
+
+          AC_DEFINE(USE_HOME_ETC,1,
+                  [ Define if want to use the HOME-ETC library. ])
+          need_home_etc=yes
+        fi
+        ])
+AM_CONDITIONAL(USE_HOME_ETC, test x$need_home_etc = xyes)
+
+
+
+
+LIBS="$LIBS $ELMOLIBS"
+
 AC_OUTPUT(Makefile \
 	  po/Makefile.in \
 	  po/Makefile \
diff -Nur elmo-1.0.4.orig/src/elmo.c elmo-1.0.4/src/elmo.c
--- elmo-1.0.4.orig/src/elmo.c	2004-01-19 11:06:10.000000000 +0100
+++ elmo-1.0.4/src/elmo.c	2004-03-01 17:42:49.000000000 +0100
@@ -82,6 +82,14 @@
 #include "procmail.h"
 #include "smtp.h"
 
+#ifdef HAVE_HOME_ETC_H
+# include <home_etc.h>
+# define HOMEDIR_RC _HEdir
+#else
+# define HOMEDIR_RC getenv("HOME")
+#endif
+
+
 /****************************************************************************
  *    IMPLEMENTATION PRIVATE DEFINITIONS / ENUMERATIONS / SIMPLE TYPEDEFS
  ****************************************************************************/
@@ -227,7 +235,7 @@
         char *file;
 
         file = (config_file) ? xstrdup (config_file)
-                : file_with_dir (getenv ("HOME"), ".elmorc");
+                : file_with_dir (HOMEDIR_RC, ".elmorc");
 
         if (confread_read_file (file) == -1){
                 confread_read_file (SYSCONFDIR "/elmorc");
